app/models/member.rb

```diff
@@ -0,0 +1,5 @@
+class Member < ActiveRestClient::Base
```

Comment:
It looks like this class generally handles the Meetup Group, rather than just a specific Member of the group. Could you rename it to something like `MeetupGroup` for clarity?